src/plugins/plugin.legend.js

```diff
@@ -96,6 +96,12 @@ module.exports = function(Chart) {
 			labelOpts.boxWidth;
 	}
 
+	function getBoxHeight(labelOpts, fontSize) {
+		return labelOpts.boxHeight ?
```

Comment:
If we consider `0` a valid height (which I think is), then:
```javascript
return helpers.isNullOrUndef(options.boxHeight) ? options.boxHeight : fontSize;
```
else this can be simplified as:
```javascript
return options.boxHeight || fontSize;
```

Reply:
Do we need to handle `labelOpts.usePointStyle` as in `getBoxWidth()`? or is it unrelated?